util/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ set(C_FILES
 if (HAVE_COHERENT_DMA)
   publish_internal_headers(util
     mmio.h
+    s390_mmio_insn.h
```

Comment:
Keep sorted please

Reply:
@jgunthorpe I'm confused, this is alphabetically sorted. First letters are "m", "s", "u"?

Reply:
Sorry I'm a little confused. the `mmio.h`, `s390_mmio_insn.h`, and `udma_barrier.h` are alphabetically sorted and `util` is just the destination so needs to be the first argument to `publish_internal_headers()`. What am I missing?

Reply:
Opps my mistake, it is right